src/main/java/controllers/GameBridge.java

```diff
@@ -0,0 +1,10 @@
+package controllers;
+
+import enums.Rank;
```

Comment:
I'm a bit confused on where in CA the enums live, as they represent parts of the business logic, but are also used in lower level classes. Maybe an exception could be made for this, I'm not too sure.

Reply:
i believe theyre stable enough to be used as another data structure for this program, like custom ints or floats